### ISSUE TITLE
Whitespace differences are hard to identify

### ISSUE THREAD

**Issue by PurpleBooth** [specdown/specdown#3]
For example

```
- verify output from '1' failed
+++ expected:
Demonstration Commit Message
 
This is a commit message that has trailers and is invalid

Signed-off-by: Someone Else <someone@example.com>
Signed-off-by: Someone Else <someone@example.com>
Co-authored-by: Billie Thompson <billie@example.com>
Co-authored-by: Billie Thompson <billie@example.com>


---

Your commit message has duplicated trailers

You can fix this by deleting the duplicated "Signed-off-by", "Co-authored-by" fields

+++ got:
Demonstration Commit Message

This is a commit message that has trailers and is invalid

Signed-off-by: Someone Else <someone@example.com>
Signed-off-by: Someone Else <someone@example.com>
Co-authored-by: Billie Thompson <billie@example.com>
Co-authored-by: Billie Thompson <billie@example.com>


---

Your commit message has duplicated trailers

You can fix this by deleting the duplicated "Signed-off-by", "Co-authored-by" fields

```

**Comment by tomphp** [comment]
I'm thinking one fo these libraries might help https://crates.io/search?page=1&per_page=10&q=diff%20color

**Comment by PurpleBooth** [comment]
I use this one for my git highlighting https://github.com/mookid/diffr

**Comment by PurpleBooth** [comment]
#4

**Comment by tomphp** [comment]
Closing because fixed by #4
